lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -174,6 +174,9 @@ class Serializer {
 
   bool isVoidType(Type type) const { return type.isa<NoneType>(); }
 
+  /// Method that checks if a type is suitable for a resource variable.
```

Comment:
Nit:

/// Returns true if the type is suitable for a resource variable.

Reply:
Thanks! Fixed.